src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { Component, ChangeEvent } from 'react';
+import React, { Component, ChangeEvent, forwardRef } from 'react';
 import { default as styled } from 'styled-components';
 import classnames from 'classnames';
 import { InputStatus } from '../types';
 import {
-  HtmlInput,
   HtmlInputProps,
   HtmlDiv,
   HtmlDivProps,
+  HtmlInputWithRef,
```

Comment:
Same comment for this as for the HtmlButton. Should we just drop the HtmlInput without ref and rename the withRef to match the original? Btw. there was a typing issue with the original input with ref, has this been fixed?